app/views/guidance/about-register-trainee-teachers.html

```diff
@@ -0,0 +1,96 @@
+{% extends "_templates/_guidance-page.html" %}
+
+{% set pageHeading = 'About Register trainee teachers' %}
+
+{% block guidanceContent %}
+
+# About Regsiter trainee teachers
+
+Register trainee teachers (Register) is the service that will replaced the Database of Trainee Teachers and Providers (DTTP) portal in 2022.
+
+Initial Teacher Training (ITT) providers use Register to provide the Department for Education (DfE) with their trainee data.
+
+The DfE uses trainee data in Register to:
+* allocate the correct amount of funding to training providers and eligible trainees
+* create the ITT census publication
+* create the ITT performance profiles
+* inform policy development
+
+## Register access for accredited providers
+
+Accredited providers like school centered initial teacher training providers (SCITTs) and higher education institutions (HEIs) can perform tasks for their trainees in Register, these include:
+
+* withdrawing or deferring trainees from their course
+* getting a teacher reference number (TRN)
+* recommending trainees for qualified teacher status (QTS) or early years teacher status (EYTS)
+
+If you have eligible trainees for funding, you can view funding information from the 2021 to 2022 academic year onwards. You can view this from the main navigation bar under ‘Funding’. Accredited providers will see monthly payments paid and predicted to be paid for scholarships and bursaries.
+
+## Register access for lead schools
+
+Lead schools that deliver school direct initial teacher training (ITT) courses in partnership with an accredited ITT body, can access Register to:
```

Comment:
```suggestion
Lead schools that deliver school direct initial teacher training (ITT) courses in partnership with an accredited provider, can access Register to:
```

Should this match how it's referred to above?